docs/02-guides/server-side-events/create-an-integration.mdx

```diff
@@ -7,7 +7,135 @@ sidebar_position: 1
 
 <p>{frontMatter.description}</p>
 
-:::info WIP
+This guide will help you to create an integration that will dispatch events to external services.
+
+We will create a new integration to dispatch events to [Wehook.cool](https://webhook.cool/).
+
+:::info
+
+Front-Commerce already provide a Webhook integration. This guide will show you how to create an integration from scratch. And will explain you the basics of integrations so you can write your own
+
+:::
+
+## Create an integration file
+
+First you need to create an integration file let's say we will call our integration Coolwebhook
+
+Let's create a new file in `app/server-evets/integrations/CoolWebhookIntegration.ts`
+
+Integrations are classes that extends `Integration interface`, let's make a simple copy paste of this code:
+
+```ts title="app/server-evets/integrations/CoolWebhookIntegration.ts"
+import type { Integration } from "@front-commerce/core/server-events/integrations";
+
+export default class CoolWehbookIntegration implements Integration {
+  compatibleEvents: string[] | "*" = "*";
+
+  webhookId: string;
+
+  constructor(webhookId: string) {
+    this.webhookId = webhookId;
+  }
+
+  handle(messageId: string, payload: any) {
+    console.log(messageId);
+    console.log(payload)
+  }
+}
+```
+
+In this class, we have four entities :
+- `compatibleEvents`: this indicates which events the integration will handle, it can be `*` to handle all events or an array of string that contains compatible events.
+- `webhookId`: this is the id of WebhookCool.
+- `constructor`: it will allow us to implement logic when class is created, here, we will store the webhookId of WebhookCool into our class.
+- `handle`: this is the method that will be called when an event is received.
+
+For your integrations, only `compatibleEvents` and `handle` are required, the `webhookId` and the constructor is an example of custom method and properties that you can implement into your integrations.
+
+## Write specific code for your integration
+
+All the event processing part will be done in the `handle` method. So we need to write all the code to send event to external services here.
+
+Since Webhook.cool is a basic services, we ca use a simple fetch to send the event :
+
+```ts title="app/server-evets/integrations/CoolWebhookIntegration.ts"
+import type { Integration } from "@front-commerce/core/server-events/integrations";
+
+export default class CoolWehbookIntegration implements Integration {
+  compatibleEvents: string[] | "*" = "*";
+
+  webhookId: string;
```

Comment:
Here you can add a comment instead of the previous documentation
```suggestion
  webhookId: string; // this is the id of the unique WebhookCool url to use (e.g: `hissing-dentist-09` for https://webhook.cool/at/hissing-dentist-09)
```